contracts/utils/RegistryBootstrap.sol

```diff
@@ -0,0 +1,177 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.26;
+
+// ──────────────────────────────────────────────────────────────────────────────
+//     _   __    _  __
+//    / | / /__ | |/ /_  _______
+//   /  |/ / _ \|   / / / / ___/
+//  / /|  /  __/   / /_/ (__  )
+// /_/ |_/\___/_/|_\__,_/____/
+//
+// ──────────────────────────────────────────────────────────────────────────────
+// Nexus: A suite of contracts for Modular Smart Accounts compliant with ERC-7579 and ERC-4337, developed by Biconomy.
+// Learn more at https://biconomy.io. For security issues, contact: security@biconomy.io
+
+import { ModuleManager } from "../base/ModuleManager.sol";
+import { IModule } from "../interfaces/modules/IModule.sol";
+import { IERC7484 } from "../interfaces/IERC7484.sol";
+
+/// @title Bootstrap Configuration for Nexus
+/// @notice Provides configuration and initialization for Nexus smart accounts.
+/// @author @livingrockrises | Biconomy | chirag@biconomy.io
+/// @author @aboudjem | Biconomy | adam.boudjemaa@biconomy.io
+/// @author @filmakarov | Biconomy | filipp.makarov@biconomy.io
+/// @author @zeroknots | Rhinestone.wtf | zeroknots.eth
+/// Special thanks to the Solady team for foundational contributions: https://github.com/Vectorized/solady
+struct BootstrapConfig {
+    address module;
+    bytes data;
+}
+
+/// @title Bootstrap
+/// @notice Manages the installation of modules into Nexus smart accounts using delegatecalls.
+contract Bootstrap is ModuleManager {
```

Comment:
If we assume this will be used on all factories and part of Nexus, can we merge the `RegistryAdapter` into `ModuleManager` and move the state variable to Storage.sol?

Reply:
different contracts are good for separation of concern.

Bootstrap could be is ModuleManager, RegistryAdapter (keeping registry storage in adapter)

or maybe just moving storage to Storage.sol